scripts/microbe_stage/microbe.as

```diff
@@ -433,14 +434,17 @@ class MicrobeSystem : ScriptSystem{
             microbeComponent.movementFactor =  microbeComponent.movementFactor/ENGULFING_MOVEMENT_DIVISION;
         }
 
+    microbeComponent.escapeInterval = microbeComponent.escapeInterval + logicTime;
```

Comment:
The indent level doesn't match up here and a couple of other spots.

Reply:
Will do but then the format script hates it

Reply:
Ah think I have found the problem